detector/write_wrong_detections.py

```diff
@@ -0,0 +1,22 @@
+# save wrong detections to folders for further analysis
+import os
+import shutil
+from torchvision.utils import save_image
+
+
+def write_wrong_detections(data_path, x_test, y_test, is_correct):
+    if data_path + "wrong_defect":
```

Comment:
I think what you want is [this](https://www.geeksforgeeks.org/python-os-path-exists-method/), right? Otherwise the code will be always executed, quite dangerous for `shutil.rmtree`
```suggestion
    if os.path.exists(data_path + "wrong_defect"):
```

Reply:
Used os.path.exists in newer version